test/specs/reference/tests/10.update.js

```diff
@@ -110,6 +113,138 @@ exports.execute = function (options) {
             });
         });
 
+        describe("for updating asset columns, ", function() {
+            var reference, column,
+                columnName = "uri",
+                tableName = "file_update_table",
+                sortBy = "key",
+                baseUri = options.url + "/catalog/" + catalogId + "/entity/" + schemaName + ':' + tableName + "/key=1";
+
+            var file = {
+                name: "testfile500kb.png",
+                size: 512000,
+                displaySize: "500KB",
+                type: "image/png",
+                hash: "4b178700e5f3b15ce799f2c6c1465741",
+                hash_64: "SxeHAOXzsVznmfLGwUZXQQ=="
+            }
+
+            beforeAll(function (done) {
+                var filePath = "test/specs/reference/files/" + file.name;
+
+                exec("perl -e 'print \"\1\" x " + file.size + "' > " + filePath);
+
+	        	file.file = new File(filePath);
+
+                options.ermRest.resolve(baseUri, { cid: "test" }).then(function (response) {
+                    reference = response;
+                    reference = reference.contextualize.entryEdit;
+
+                    column = reference.columns.find(function(c) { return c.name == columnName;  });
+
+                	if (!column) {
+                		console.log("Unable to find column " + columnName);
+                		done.fail();
+                		return;
+                	}
+                    done();
+                }, function (err) {
+                    console.dir(err);
+                    done.fail();
+                });
+            });
+
+            it("the metadata for the asset should be updated.", function (done) {
```

Comment:
I added a note of this in the e2e dev guide in the wiki. This is actually something I learned from @chiragsanghvi test cases and something we need to do more often going forward.